src/components/App.js

```diff
@@ -0,0 +1,77 @@
+import React, { Fragment, PureComponent } from 'react';
+import '../css/App.css';
+import domToImage from 'dom-to-image';
+import fileSaver from 'file-saver';
+import MemeForm from './MemeForm';
+import ImageFormatter from './ImageFormatter';
+import 'normalize-css';
+
+export default class App extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.memeRef = React.createRef();
+  }
+
+  state = {
+    header: '', 
+    footer: '',
+    imageUrl: '',
+    headerColor: '',
```

Comment:
```suggestion
    headerColor: '#000000',
```
Because you set the input type to color, the initial value of the header and footer colors should be set to something like #000000.